CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ Current
 -------
 ### Fixed:
 
+- [Fix: Presto Limit](https://github.com/yahoo/fili/pull/1128)
+
```

Comment:
* Create a github issue and link to it for the feature.
* Make note of the tests that you're ignoring and removing and why (flaky on the first and non-compiling on the second)

Reply:
Sure. Created https://github.com/yahoo/fili/issues/1129